source/renderer/app/App.js

```diff
@@ -30,13 +31,18 @@ export default class App extends Component<{
   }
   render() {
     const { stores, actions, history } = this.props;
-    const { app } = stores;
+    const { app, nodeUpdate, networkStatus } = stores;
+    const { isConnected, isSynced } = networkStatus;
     const { isActiveDialog } = app;
     const locale = stores.profile.currentLocale;
     const mobxDevTools = global.environment.mobxDevTools ? <DevTools /> : null;
     const { currentTheme } = stores.profile;
     const themeVars = require(`./themes/daedalus/${currentTheme}.js`).default;
     const { ABOUT, BLOCK_CONSOLIDATION, DAEDALUS_DIAGNOSTICS } = DIALOGS;
+    const { isUpdateAvailable, isUpdatePostponed } = nodeUpdate;
+
+    const showAutomaticUpdateDialog =
```

Comment:
@tomothespian please move this logic into the store. Probably the `nodeUpdate` one is the best option...

Reply:
Also, I am not sure if we even need to check for `isConnected ` or `isSynced`. Can you try to leave them out?